### PR TITLE
Pulling most recent files in R -- jsandrad

### PR DESCRIPTION
**Pull request by jsandrad**
@naupaka @melt6457 @mhaliz

### CODE REVIEW

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Pull most recent files in R"
+date:   2020-11-09
+categories: jekyll update
```
Comment (jordanseaton) [inline _posts/2020-11-09-pulling-most-recent-files.markdown]: "Categories" should include tags related to your blog post. You could change this from “jekyll update” to “R” and any other related tags.

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+---
+# Pulling most recent files in R
+ ###### by Joel Christian Andrade
+
+Many coding pipelines involve taking raw data files, formulating code and eventually working with a few, if not many, output files.
+
+Take for example the list of files, in my directory `/raw_data` below from the publicly available Apple mobility database tracking movement.
```
Comment (jordanseaton) [inline _posts/2020-11-09-pulling-most-recent-files.markdown]: I get what you're trying to say here, but I think this sentence could be reworded to make it a little easier to read. Something like "Take for example this list of files from my `/raw_data` directory, which were downloaded from the publicly available Apple mobility database."

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+applemobilitytrends_2020-08-29.csv
+applemobilitytrends_2020-09-12.csv
+applemobilitytrends_2020-09-18.csv
+applemobilitytrends_2020-09-25.csv
+applemobilitytrends_2020-09-30.csv
+applemobilitytrends_2020-10-02.csv
+applemobilitytrends_2020-11-08.csv
+```
+
+While retroactive data analysis is useful, the most recent file will include the data of the previous files.  Thus, to retrieve the most up to date information while using R, we must use some functions which are pre-loaded into R.
```
Comment (jordanseaton) [inline _posts/2020-11-09-pulling-most-recent-files.markdown]: I suggest rewording this to say "the most recent file will include all data from the previous files"

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+   * `mtime` = modification time
+   * `atime` = access time
+   * `ctime` = creation time
+
+
+* `list.files` = list of all files within a directory
+
+* `which.min` or `which.max` = determine minimum or maximum with respect to a given set of variables
+
+Let's take the directory `raw_data` and view various outputs using these functions
```
Comment (jordanseaton) [inline _posts/2020-11-09-pulling-most-recent-files.markdown:43]: I like how you included a detailed example!

**Review by jordanseaton — COMMENTED** [review]
@jsandrad Great job on your blog post! I’ve included a few suggestions that may help get some of your points across more effectively.

**Comment by jsandrad** [comment]
Got the fixes you noted!

File: _posts/2020-11-09-pulling-most-recent-files.markdown (hunk 1 above)
Comment (jsandrad) [reply _posts/2020-11-09-pulling-most-recent-files.markdown]: changed to R base packages

File: _posts/2020-11-09-pulling-most-recent-files.markdown (hunk 2 above)
Comment (jsandrad) [reply _posts/2020-11-09-pulling-most-recent-files.markdown]: reworded

File: _posts/2020-11-09-pulling-most-recent-files.markdown (hunk 4 above)
Comment (jsandrad) [reply _posts/2020-11-09-pulling-most-recent-files.markdown:43]: thanks!

File: _posts/2020-11-09-pulling-most-recent-files.markdown (hunk 3 above)
Comment (jsandrad) [reply _posts/2020-11-09-pulling-most-recent-files.markdown]: rephrased

**Comment by melt6457** [comment]
@jordanseaton Nice work on the thoughtful review!!

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+              size  isdir  mode              mtime                ctime                atime  exe  
+data/raw_data    0   TRUE   777 2020-10-4 13:03:40  2020-09-19 17:14:40  2020-10-04 13:03:40  no
+```
+
+`list.files()`
+```r
+> list.files("/raw_data")
+[1] "applemobilitytrends_2020-09-12.csv" "applemobilitytrends_2020-09-18.csv" "applemobilitytrends_2020-09-25.csv"
+[4] "applemobilitytrends_2020-09-30.csv" "applemobilitytrends_2020-10-02.csv" "applemobilitytrends_2020-08-29.csv"
+```
```
Comment (melt6457) [inline _posts/2020-11-09-pulling-most-recent-files.markdown]: @jsandrad maybe it would be easier on the reader if these files matched the ones above?

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+applemobilitytrends_2020-09-12.csv
+applemobilitytrends_2020-09-18.csv
+applemobilitytrends_2020-09-25.csv
+applemobilitytrends_2020-09-30.csv
+applemobilitytrends_2020-10-02.csv
+applemobilitytrends_2020-11-08.csv
+```
+
+While retroactive data analysis is useful, the most recent file will include all data from previous files.  Thus, to retrieve the most up to date information while using R, we must use some functions which are pre-loaded into R.
+
```
Comment (melt6457) [inline _posts/2020-11-09-pulling-most-recent-files.markdown:27]: @jsandrad consider adding another header here...something like `Useful R Functions` or `R functions we will be using`

File: _posts/2020-11-09-pulling-most-recent-files.markdown [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+`list.files()`
+```r
+> list.files("/raw_data")
+[1] "applemobilitytrends_2020-09-12.csv" "applemobilitytrends_2020-09-18.csv" "applemobilitytrends_2020-09-25.csv"
+[4] "applemobilitytrends_2020-09-30.csv" "applemobilitytrends_2020-10-02.csv" "applemobilitytrends_2020-08-29.csv"
+```
+
+
```
Comment (melt6457) [inline _posts/2020-11-09-pulling-most-recent-files.markdown:62]: Maybe another header like `Putting it all together`

**Review by melt6457 — CHANGES_REQUESTED** [review]
@jsandrad great idea on this blog post and good execution! I think it would be helpful to add some headers and ensure the files match throughout the post!

**Comment by jsandrad** [comment]
added suggestions to most recent commit @melt6457

**Review by melt6457 — APPROVED** [review]
@jsandrad Nice work on these changes!!
